src/features/search/components/SearchBox.tsx

```diff
@@ -21,16 +22,21 @@ const LeftIcon: React.FC<{ onPressArrowBack: () => void }> = ({ onPressArrowBack
   const { searchState } = useSearch()
   if (searchState.showResults)
     return (
-      <TouchableOpacity onPress={onPressArrowBack}>
+      <TouchableOpacity
+        onPress={onPressArrowBack}
+        {...accessibilityAndTestId(t`Revenir en arrière`)}>
         <ArrowPrevious />
       </TouchableOpacity>
     )
-  return <MagnifyingGlass />
+  return <MagnifyingGlass {...accessibilityAndTestId(t`Rechercher`)} />
```

Comment:
Si c'est pas touchable, pour le moment on ne met pas d'accessibility
```suggestion
  return <MagnifyingGlass />
```